WorkflowSubmission/Program.cs

```diff
@@ -32,11 +32,11 @@ async static Task Main(string[] args)
             // Submit files to Workflow
             WorkflowSubmission workflowSubmission = new WorkflowSubmission(client)
             {
-                Id = workflow.Id,
+                WorkflowId = workflow.Id,
                 Files = FILE_LIST
             };
 
-            Job job = await workflowSubmission.Exec();
+            List<int> submissionIds = await workflowSubmission.Exec();
```

Comment:
How do we get the resulting JSON from the submission id?

Reply:
Can you please provide an example of how to get results from the submission id?  Kind of equivalent to `JArray jobResult = await job.Results();`